weather-app/helpers/getWeather.js

```diff
@@ -0,0 +1,31 @@
+const request = require('request')
+
+const getWeather=(lat,long,callback)=>{
+
+    const url ="https://api.darksky.net/forecast/e48d111db62f38b9ae1c9cad16f7b92b/"+lat+","+long
+
+    request(url, (error, response) => {
+        if (error) {
+            callback(error, null)
```

Comment:
please test this by shooting down internet connection (Code will be moving forward)